pyuvsim/simsetup.py

```diff
@@ -746,7 +747,8 @@ def parse_frequency_params(freq_params):
                 raise ValueError("Either channel_width or Nfreqs "
                                  " must be included in parameters:" + kws_used)
             if sf and ef:
-                freq_params['bandwidth'] = freq_params['end_freq'] - freq_params['start_freq'] + freq_params['channel_width']
+                freq_params['bandwidth'] = freq_params['end_freq'] - freq_params['start_freq'] + freq_params[
+                    'channel_width']
```

Comment:
same as line-breaking comment

Reply:
Fixed in b7e89d70fab47c00273dbfa95fa5f5c82f3af8b0